### ISSUE TITLE
Named anchor not working in internal / external preview

### ISSUE THREAD

**Issue by ajit555** [RickStrahl/MarkdownMonster#414]
Trying to create a Table of Content (TOC) in markdown using named anchor.

```markdown
<!--Creating table of content using named anchor-->
* Take me to [pookie](#pookie)

<!--which links to -->
### <a name="pookie"></a>This is the Heading for Pookie
```

When I save the document as HTML, in html file, the named anchor tag is working but not in the MM's internal / external document preview.

**Comment by ajit555** [comment]
Found the MM TOC generator add-in, which works for me.

**Comment by RickStrahl** [comment]
There's a bug in the hash resolution. You shouldn't need the TOC addin to make this work. Fixed in 22b203b59df80dd6f936fd7ebc17081681553ee5. Maybe give this another try after this is applied in 1.12.11 and later updates.
